CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contribution Guidelines
+
+If you're planning to contribute, **please follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard.**
+
+Pull Requests (PRs) that do not adhere to this standard will **not be accepted**.
+
+## Branch Requirements
+
+- **All PRs must be based on the `dev` branch.**
+- Merges will only occur into the `dev` branch before being released to the main branch.
```

Comment:
...

Reply:
I am above the rules

Reply:
no you are not :eyes: 
in the dev branch with this 💩

Reply:
It's a draft for a reason my guy

Reply:
good boy